### ISSUE TITLE
Try to add more example related to "Query Conditions"

### ISSUE THREAD

**Issue by byanjiong** [go-ozzo/ozzo-dbx#56]
The intro page provides limited examples on the query conditions, as a newbie, I don't really know how to use it for complex conditions...

How about adding more examples? E.g.:

```
// id = 100 OR id = 73
// id = 200 AND (status="publish" OR status="edit")
// id != 100 OR ( name IN("james", "robert", "mike") )
```

something like this will help us to construct complex conditions easily... ^_^

and possible provide some examples related to timestamp operation, e.g. `less than 24 hours`, `more than 3 days`...

I can write plain SQL statement, however, when trying to use function to wrap it, then hardly have idea how to get it done... Hoping for some (more complex and real-life) examples...

Sorry for asking a newbie question... 抱歉... :)

Thanks...

**Comment by kolkov** [comment]
Agreed, the docs could use more examples for complex conditions. Something like:

```go
// id = 200 AND (status='publish' OR status='edit')
dbx.And(dbx.HashExp{"id": 200}, dbx.Or(dbx.HashExp{"status": "publish"}, dbx.HashExp{"status": "edit"}))
```

We'll keep this in mind for a documentation update. This has been sitting for a while though — still needed? We'll close in a couple of weeks if no reply.

**Comment by kolkov** [comment]
Closing — the And/Or composition example provided above should cover most query condition patterns. Reopen if needed.
